web_view_calendar_list/static/src/js/calendar_list_renderer.js

```diff
@@ -0,0 +1,154 @@
+                    self.trigger_up('dropRecord', event);
+                },
+                eventResize: function (event) {
+                    self.trigger_up('updateRecord', event);
+                },
+                eventClick: function (event) {
+                    self.trigger_up('openEvent', event);
+                    self.$calendar.fullCalendar('unselect');
+                },
+                select: function (target_date, end_date, event, _js_event, _view) {
+                    var data = {'start': target_date, 'end': end_date};
+                    if (self.state.context.default_name) {
+                        data.title = self.state.context.default_name;
+                    }
+                    self.trigger_up('openCreate', data);
+                    self.$calendar.fullCalendar('unselect');
+                },
+                eventRender: function (event, element) {
+                    var $render = $(self._eventRender(event));
+                    event.title = $render.find('.o_field_type_char:first').text();
+                    element.find('.fc-list-item-title').html($render.html());
+                    element.addClass($render.attr('class'));
+                    var display_hour = '';
+                    if (!event.allDay) {
+                        var start = event.r_start || event.start;
+                        var end = event.r_end || event.end;
+                        var timeFormat = _t.database.parameters.time_format.search("%H") != -1 ? 'HH:mm': 'h:mma';
+                        display_hour = start.format(timeFormat) + ' - ' + end.format(timeFormat);
+                        if (display_hour === '00:00 - 00:00') {
+                            display_hour = _t('All day');
+                        }
+                    }
+                    element.find('.fc-list-item-time').text(display_hour);
+                },
+                // Dirty hack to ensure a correct first render
+                eventAfterAllRender: function () {
+                    $(window).trigger('resize');
+                },
+                viewRender: function (view) {
+                    // compute mode from view.name which is either 'month', 'agendaWeek' or 'agendaDay'
+                    var mode = view.name === 'listMonth' ? 'month' : (view.name === 'listWeek' ? 'week' : 'day');
+                    // compute title: in week mode, display the week number
+                    var title = mode === 'week' ? view.intervalStart.week() : view.title;
+                    self.trigger_up('viewUpdated', {
+                        mode: mode,
+                        title: title,
+                    });
+                },
+                height: 'parent',
+                unselectAuto: false,
+                locale: locale, // reset locale when fullcalendar has already been instanciated before now
+            });
+
+            this.$calendar.fullCalendar(fc_options);
+        },
+
+        _render: function () {
```

Comment:
It's a pity that you need to override the whole function just to change the last line to prevent it from calling _super but there is no other option right?

Reply:
@etobella can you explain w/ a detailed comment?

Reply:
The main reason why we need to overwrite all is the variables `scales`. It is defined as an external variable undependant on the class, so, in order to modify it, we need to overwrite all the function :sob:

Reply:
I will add the comment on the JS

Reply:
Maybe you can modify methods where '_render' is called and pass as argument the new "scales". "Hoisting" do the rest of the magic... :/